sql/sql_class.cc

```diff
@@ -7670,7 +7685,6 @@ wait_for_commit::wait_for_prior_commit2(THD *thd)
   remove_from_list(&loc_waitee->subsequent_commits_list);
   mysql_mutex_unlock(&loc_waitee->LOCK_wait_commit);
   this->waitee.store(NULL, std::memory_order_relaxed);
-
```

Comment:
Unnecessary white spaces removal here and elsewhere.
Could you please restore those hunk to their BASE.